test/options-handler.js

```diff
@@ -346,8 +346,7 @@ describe('Options handler', () => {
         const libs = optionsHandler.parseLibraries({fake: optionsProps.libs});
         const compilerInfo = makeFakeCompilerInfo('g82', 'c++', 'cpp', '8.2', true);
         const env = {
-            ceProps: () => {
-            },
+            ceProps: (_, defaultValue) => defaultValue,
```

Comment:
as this is so common I'd be tempted to teach `lib/properties/fakeProps` how to do this and use that here (if it does'nt already work out of the bag?)